internal/backends/android-activity/androidwindowadapter.rs

```diff
@@ -224,6 +224,13 @@ impl AndroidWindowAdapter {
                         None,
                     )?;
                     self.resize();
+
+                    // FIXME: I don't know why this is the case, but without this, the soft input
+                    // always prompt out on startup.
```

Comment:
```suggestion
                    // Without this, the soft input always prompt out on startup.
```

Is it only on some version of android? 
Is it because there is a text field with the focus on startup?

Reply:
Oh sorry, I tested an application without any text box, without this modification, the virtual keyboard isn't shown on Android 11.0, but it's shown on Android 6.0 and 8.0. I haven't tested Android 9.0 and 10.0 yet.

Reply:
Oh sorry, I tested an application without any text box, without this modification, the virtual keyboard isn't shown on Android 11.0, but it's shown on Android 6.0 and 8.0. I haven't tested Android 9.0 and 10.0 yet.